src/views/default/panels/db/detail.php

```diff
@@ -113,25 +128,25 @@
 ?>
 
 <script>
-function debug_db_detail() {
-    $('.db-explain a').on('click', function(e) {
-        e.preventDefault();
-        
-        var $explain = $('.db-explain-text', $(this).parent().parent());
-
-        if ($explain.is(':visible')) {
-            $explain.hide();
-            $(this).text('[+] Explain');
-        } else {
-            $explain.load($(this).attr('href')).show();
-            $(this).text('[-] Explain');
-        }
-    });
-
-    $('#db-explain-all a').on('click', function(e) {
-        e.preventDefault();
-        
-        $('.db-explain a').click();
-    });
-}
+    function debug_db_detail() {
```

Comment:
That looks jQuery-dependent. Would be great to have it re-done with plain JavaScript as well.

Reply:
fixed
I created an asset to keep the standard (like timeline or user switch)